Move and update setup files (#697)
AlexsLemonade/training-modules: scRNA-seq-advanced/setup/README.md

> Libraries were generated following the Chromium Next GEM Single Cell 3ʹ Reagent Kits v3.1 (Dual Index) User Guide (CG000315) and sequenced on Illumina NovaSeq 6000.

To download the files, change directories to the `setup/glioblastoma-10x` directory and run:

```sh
snakemake -j2
```

> This will place the downloaded files in `/shared/data/training-modules/scRNA-seq-advanced/data/glioblastoma-10x`

Reviewer: ```suggestion
This will place the downloaded files in `/shared/data/training-modules/scRNA-seq-advanced/data/glioblastoma-10x`.
```

> This dataset is also used in the `scRNA-seq/06-celltype_annotation.Rmd` notebook, but this requires it to be processed before use, so the workflow will also run the `scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd` notebook and copy the output RDS from that run to the `/shared/data/training-modules/scRNA-seq/data/glioblastoma-10x` directory.

Reviewer: ```suggestion
Because a processed version of this dataset is also used in the `scRNA-seq/06-celltype_annotation.Rmd`, this workflow will also run the `scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd` notebook (which performs this processing) and copy its `RDS` output to the `/shared/data/training-modules/scRNA-seq/data/glioblastoma-10x` directory.
```
